File: Beginner/Multiples_1044/Main.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String values = sc.nextLine();
        String[] splittedValues = values.split(" ");
        int a = Integer.parseInt(splittedValues[0]);
        int b = Integer.parseInt(splittedValues[1]);

        if (a % b == 0 || b % a == 0) {
            System.out.println("Sao Multiplos");
        }
        else {
            System.out.println("Nao sao Multiplos");
        }
        sc.close();
    }
}

File: Beginner/Multiples_1044/main.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
	"strings"
)

func main() {
	values := bufio.NewReader(os.Stdin)
	line, _ := values.ReadString('\n')
	line = strings.Trim(line, "\n")
	valuesSplitted := strings.Split(line, " ")
	a, _ := strconv.ParseInt(valuesSplitted[0], 10, 64)
	b, _ := strconv.ParseInt(valuesSplitted[1], 10, 64)
	if a%b == 0 || b%a == 0 {
		fmt.Println("Sao Multiplos")
	} else {
		fmt.Println("Nao sao Multiplos")
	}
}
